sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/aad/webapi/AADJwtBearerTokenAuthenticationConverterTest.java

```diff
@@ -18,13 +21,17 @@
 
 public class AADJwtBearerTokenAuthenticationConverterTest {
 
-    private Jwt jwt = mock(Jwt.class);
-    private Map<String, Object> claims = new HashMap<>();
-    private Map<String, Object> headers = new HashMap<>();
+    private Jwt jwt;
+    private Map<String, Object> claims;
+    private Map<String, Object> headers;
     private JSONArray jsonArray = new JSONArray().appendElement("User.read").appendElement("User.write");
 
     @BeforeEach
```

Comment:
Can we change to `@BeforeAll`?

Reply:
Yes, but the method should be changed to static and these variables also need to change, I think the non-static will be better, how about you?

Reply:
Maybe we do not need to change the variables?